CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [v1.4.0]
```

Comment:
Should we pre-emptively add a new `Unreleased` header above this? I think that's what we do on the main Cumulus repo

Reply:
Possibly.  We tend to immediately release changes on this repo, so I could go either way.